syncer/syncer.go

```diff
@@ -775,7 +780,11 @@ func (s *Syncer) updateReplicationLag(job *job, queueBucketName string) {
 		s.secondsBehindMaster.Store(lag)
 	}
 	// when job is nil and s.jobs is emepty, means all event is consumed,we update lag to 0
+	// also neet do reset al workerLagMap log to 0
 	if job == nil && s.jobsIsEmpty() {
+		for _, l := range s.workerLagMap {
+			l.Store(0)
+		}
```

Comment:
Don't need `s.jobsIsEmpty()`. We can set `s.workerLagMap` for each bucket.

Reply:
we still need `s.jobsIsEmpty()` beause when one job bucket is empyt thiere is may some job running in other bucket

so i only update passed queueBucketName's  lag to 0 , and only update metrics to 0 when `s.jobsIsEmpty()`